.yarn/versions/09625992.yml

```diff
@@ -1,6 +1,3 @@
-releases:
```

Comment:
Was there a need to modify these semver docs? Usually it's not something that's manually changed.